plugins/ui/src/deephaven/ui/renderer/Renderer.py

```diff
@@ -61,6 +61,8 @@ def _render_item(item: Any, context: RenderContext) -> Any:
         The rendered item.
     """
     logger.debug("_render_item context is %s", context)
+    if isinstance(item, map):
+        return _render_map(item, context)
     if isinstance(item, (list, tuple)):
```

Comment:
Treat `map` like the other iterable types:
```suggestion
    if isinstance(item, (list, map, tuple)):
```